Src/PChecker/CheckerCore/SystematicTesting/TestingEngine.cs

```diff
@@ -370,13 +371,7 @@ private Task CreateTestingTask()
                             RandomValueGenerator.Seed += 1;
                         }
 
-                        // Increases iterations if there is a specified timeout
```

Comment:
Why is this part dropped? Is it moved somewhere else now?

Reply:
This part is moved to PCommandLine when parsing "timeout" option of "p check".
If a timeout is passed, P CLI sets iteration limit to 0.
If a timeout is passed followed by iteration limit, both timeout and iteration limit are enforced.

Reply:
Why is it set to zero and what happens when iteration limit is zero?

Reply:
@aman-goel